src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { getTabById } from '../../utils';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = getTabById(tabs, selectedTabId) || tabs[0];
```

Comment:
The return of the first tab in case of finding the desired one should be transferred to the middle of the `getTabById` function.